LazarevDV/lab_1/calculator.cpp

```diff
@@ -0,0 +1,33 @@
+#include <iostream>
+#include <conio.h>
+
+using namespace std;
+
+int main()
+{
+    setlocale(LC_ALL, "Russian");
+    long double fv, sv;
+    char sign, check;
+    cout << "Добро пожаловать в калькулятор Дениса Лазарева!\n";   
+    do { 
+    cout << "Введите два числа и знак операции:\n";
+    cin >> sv >> fv >> sign; //Здесь я сначала сделал fv, а потом sv, но в кейсе перепутал, поэтому, первое число sv, а второе fv
+    switch (sign) {
+    case '+': cout << sv << " + " << fv << " = " << sv + fv << endl; break;
```

Comment:
Extract `cout << sv << " + " << fv << " = " << sv + fv << endl;` from switch..case in order to reduce code duplication